src/components/pages/AdminDashboard/DevTools/DayComponent.js

```diff
@@ -17,6 +17,19 @@ const DayComponent = ({ day, devMode, setDevMode }) => {
     setDevMode(!devMode.isDevModeActive);
   };
 
+  const findNextDayOfWeek = selectedDay => {
```

Comment:
Just tried this out locally, it seemed to work out really solid which is awesome, great job.  I think we should rename this function for clarity though, since it's not finding the next day, but is doing something along the lines of 'setTargetDate' or 'spoofTargetDate' or something.  Thoughts?